runtime/dependency-lister/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.apache.camel.k</groupId>
+    <artifactId>camel-k-runtime-parent</artifactId>
+    <version>0.0.3-SNAPSHOT</version>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>maven-plugin</packaging>
+
+  <artifactId>camel-k-runtime-dependency-lister</artifactId>
```

Comment:
I think we can give it a more generic name and reuse it for other tasks (in the future, not necessarily now). E.g. "camel-k-runtime-meta-plugin".
I'm thinking e.g. to have a mojo that extract info on the route (to be called out-of-band respect to image build), in order to have full info about the route and then apply the optimizations we talked about in the dev mailing list. Just thinking..

Reply:
agree, feel free to rename it :)